client/js/widgets/zoomy.js

```diff
@@ -159,28 +159,28 @@ export class Zoomy extends Widget {
     return this.get('zoomedPlayers') == 'all' || this.get('zoomedPlayers').includes(playerName);
   }
 
-  async set(property, value) {
-    if(this.isZoomed() && $('body').classList.contains('edit')) {
-      if(property == 'x')
-        await this.set('zoomedX', value);
-      else if(property == 'y')
-        await this.set('zoomedY', value);
-      else if(property == 'scale')
-        await this.set('zoomedScale', value);
-      else if(property == 'rotation')
-        await this.set('zoomedRotation', value);
-      else
-        await super.set(property, value);
-    } else {
-      await super.set(property, value);
-    }
-  }
-
```

Comment:
Did you remove this on purpose? At first glance, it doesn't seem related to this commit. But so far it really only was a glance.

Reply:
No.  That was accidental.  Restored.